lib/mastodon/emoji_cli.rb

```diff
@@ -96,5 +145,22 @@ def color(green, _yellow, red)
         :red
       end
     end
+
+    def export_file_name(emoji)
+      if options[:shortcode]
+        emoji.shortcode + '.png'
+      else
+        emoji.image_file_name
+      end
+    end
+
+    def add_emoji_to_file(emoji, exported, tar)
+      emoji_file_name = export_file_name(emoji)
+      say("Adding '#{emoji.shortcode}' as '#{emoji_file_name}'...")
```

Comment:
Unnecessary output. This would maybe make sense if each step took a long time and there was a progress bar but I think in most cases this whole thing will finish very quickly.

Reply:
Yes, sorry. That's leftover debug logging 🙈